src/components/30-organisms/commercial-hero-banner/child.scss

```diff
@@ -17,56 +17,25 @@ axa-commercial-hero-banner {
     margin-right: 0;
   }
 
-  h2 {
```

Comment:
What happend to h2? Not needed?

Reply:
It's not a `<h2>` anymore. It's a `<p>` inside a `<header>`